pkg/apparmor/apparmor_linux.go

```diff
@@ -85,23 +85,17 @@ func InstallDefault(name string) error {
 		Name: name,
 	}
 
-	cmd := exec.Command("apparmor_parser", "-Kr")
-	pipe, err := cmd.StdinPipe()
+	f, err := os.Create(path.Join(profileDirectory, name))
```

Comment:
if it is for debugging purposes can we store the file under `/var`?  I don't think we should write to `/etc`.